person.rb

```diff
@@ -0,0 +1,39 @@
+class Person
+  # Constructor with name, age, and parent_permission as parameter.
+  def initialize(age, name = 'unknown', parent_permission: true)
+    # Instance vars: @id, @name, and @age.
+    @id = nil
```

Comment:
- [ ] Instead of assigning the `id` to `nil` for each object, you can assign it to a random value between two values like the following:
```ruby
@id = Random.rand(1..1000)
```

Reply:
Sure. It wasn't specified in the instructions. Don't know why I didn't think of this😆